includes/helper/class-wp-job-manager-helper.php

```diff
@@ -908,6 +992,9 @@ private function handle_product_activation_response( $response, $product_slug, $
 			self::log_event( 'license_activation_error', $event_properties );
 		} else {
 			self::log_event( 'license_activated', $event_properties );
+
+			// Clear the update cache so we can get the packages.
+			delete_site_transient( 'wpjm_helper_updates' );
```

Comment:
Should we also delete the `update_plugins` transient?
In my test I had to flush the transients after activating the license for a plugin in order to see the "update" action in the plugins page.

Reply:
Cleared in c84f82e199ef0d01581daa5771243566baad27fa